versioned_docs/version-v6.0.0/framework/pubsubV2.md

```diff
@@ -4,28 +4,41 @@ sidebar_label: Publish and subscribe
+:::note[motivation]
+Asynchronous messaging helps with decoupling publishers from consumers, as it avoids the blocking during publishing. Especially in Logic Apps workflows, where the publisher and consumer is not always synchronous.
 
-Invictus provides a solution called the PubSub, that allows Azure Service Bus to act as a message broker and interact in a publish/subscribe-approach via HTTP endpoints.
+Invictus provides a solution called the PubSub, that allows Azure Service Bus to act as a message broker and interact in a publish/subscribe-approach via HTTP endpoints; plus having Azure Blob Storage act as a claim-check provider for when messages are to big to be published directly.
 
-## Usage
-The PubSub is available as a HTTP endpoint in your LogicApp workflow.
+*🔗 See also the [Publisher-Subscriber integration pattern](https://learn.microsoft.com/en-us/azure/architecture/patterns/publisher-subscriber).*
+:::
 
-- Default topic name: `pubsubv2router`.
-- Default claim-checked messages blob container name: `invictuspubsubv2router`.
-- Subscription filter rules are assigned the same name as the subscription.
-- Default message time to live: `30 days`
-- Default subscription lock timeout: `1 minute`
+## Available endpoints
+* [`/api/Publish`](#️-publish-single-message): by sending a HTTP request with a custom content, it places a message on an Azure Service Bus topic.
+* [`/api/Subscribe`](#️-subscribe-for-messages): by sending a HTTP request with a specified Azure Service Bus topic subscription name, it response available messages.
+* [`/api/Acknowledge`](#-acknowledge-message): by sending a HTTP request with a specified message sequence number, it settles the Azure Service Bus message.
 
-## Publish single message
-The publish request content is sent as a message to the Service Bus topic. Any values in the request context will also be added to the Service Bus message application properties. If the request headers contain the `x-ms-client-tracking-id` or `x-ms-workflow-run-id` keys, their values will also be added to the application properties. A MessageID may also be passed for duplicate detection purposes, or left empty to generate a new one.
+![PubSub pseudo Logic App diagram](/images/framework/pseudo-logic-app-w-pubsub.png)
 
-If the request content length exceeds a certain size (default is `20 0000 bytes`), then the claim check pattern will be applied. The Service Bus message will be sent with an empty body, and the content saved as a blob in Azure Storage instead. Specific properties are set within the message so that the content may be retrieved when subscribing.
+## ➡️ Publish single message
+The `/api/Publish` endpoint allows users to send a single message to the configured Azure Service Bus topic (default: `pubsubv2router`) where subscribers are listening to.
 
-### Request Body Example
+The following request properties are available during publication:
```

Comment:
Might be better to rename to `The following request properties can be supplied`?

Currently it seems a bit like these properties are available but we actually need to supply them ourselves